test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Post_AppicationJson_Decimal_Tests.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests
+{
+    [TestClass]
+    [TestCategory(Constants.TestCategory)]
+    public class Post_AppicationJson_Decimal_Tests
+    {
+        private static HttpClient http = new HttpClient();
+
+        private JObject _doc;
+
+        [TestInitialize]
+        public async Task Init()
+        {
+            var json = await http.GetStringAsync(Constants.OpenApiDocEndpoint).ConfigureAwait(false);
+            this._doc = JsonConvert.DeserializeObject<JObject>(json);
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBody(string path, string operationType)
+        {
+            var requestBody = this._doc["paths"][path][operationType]["requestBody"];
+
+            requestBody.Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "text/plain")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBodyContentType(string path, string operationType, string contentType)
+        {
+            var content = this._doc["paths"][path][operationType]["requestBody"]["content"];
+
+            content[contentType].Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "text/plain", "number", "decimal")]
```

Comment:
Here is a problem. Now data row has "decimal" value in propertyFormat, but I the format of your app does not have any "decimal" format. I think it's better to change "decimal" to "double"

Reply:
Thanks, I'll give it a try